component-models.json

```diff
@@ -1554,5 +1554,55 @@
         "label": "List of Cities"
       }
     ]
+  },
+  {
+    "id": "loan-steps",
```

Comment:
Do you expect more style variations for this block ? otherwise loan-steps doesnt need a model.

Reply:
I aligned it like `testimonies` block (which will have `testimony` items added)